katharsis-servlet/src/test/java/io/katharsis/servlet/resource/repository/LocaleRepository.java

```diff
@@ -0,0 +1,42 @@
+package io.katharsis.servlet.resource.repository;
+
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import io.katharsis.queryParams.QueryParams;
+import io.katharsis.repository.ResourceRepository;
+import io.katharsis.servlet.resource.model.Locale;
+
+/**
+ * Created by nickmitchell on 1/6/17.
+ */
+public class LocaleRepository implements ResourceRepository<Locale, Long> {
```

Comment:
Also, there is already Abstract classes that implements all these methods and allows you to just override what you need. Please use that one if this class turns out to be necessary